sql/mob_pools.sql

```diff
@@ -3256,7 +3256,7 @@ INSERT INTO `mob_pools` VALUES (3200,'Processionaire','Processionaire',79,0x0000
 INSERT INTO `mob_pools` VALUES (3201,'Proconsul_XII','Proconsul_XII',25,0x0000110500000000000000000000000000000000,7,7,3,265,100,0,1,0,1,2,0,0,238,135,0,0,4,0,0,25,25);
 INSERT INTO `mob_pools` VALUES (3202,'Procreator','Procreator',138,0x00006D0400000000000000000000000000000000,1,1,3,240,100,0,0,1,0,2,1,0,6,133,0,0,0,0,0,138,138);
 INSERT INTO `mob_pools` VALUES (3203,'Procrustes','Procrustes',126,0x0000120100000000000000000000000000000000,4,4,12,240,100,0,1,1,1,2,0,0,0,3,0,0,2,0,0,126,126);
-INSERT INTO `mob_pools` VALUES (3204,'Progenerator','Progenerator',138,0x00006D0400000000000000000000000000000000,1,1,3,240,100,0,1,1,0,16,1,0,0,0,0,0,0,0,0,138,138);
+INSERT INTO `mob_pools` VALUES (3204,'Progenerator','Progenerator',138,0x00006D0400000000000000000000000000000000,1,1,3,180,100,0,1,1,0,16,5,0,0,6,0,0,0,0,0,138,138);
```

Comment:
eyeballed/best guess or sourced?
this isn't as significant a change as the overgrown ivy pr, so unless we can source or prove the numbers I'd rather leave them alone here. Anyone on LSB with a dissenting opinion, please post.

Reply:
We basically have to figure out the mobs tp gains to know its exact delay, which videos generally don't do.

Reply:
I just had an idea for an addon to maybe make this easier on contributors. cross yer fingers.

Reply:
Most delays, if not all are all "guessed"/based on videos. No wiki or any source I found actually has definitive delay values for almost any mob in the game (unless I am blind). These mobs are pretty easy as the double attack and delays are low/low chance so It's pretty accurate. Timers on auto attacks only do so much as they can vary by 1-2 seconds depending on the server lag. I can just leave them out for now unless we get something for these values. I spent hours staring at multiple videos staring at timers/animation delays.

I will take any advice/help if it makes this easier in the future. It's beyond my knowledge on how to do it otherwise.

Reply:
Most of the extant mobs in our DB use known values for common categories. mnk h2h base delay without a weapon equipped is known (and martial arts applies after, outside the pools table), most other things use a 240 standard, SE uses several typical ones but 240 is the most common which is why so many mobs have that by default. 224 is another one occasionally used. Nin mobs that dual wield (there are both single and dual wield ones) often have a higher delay similar to h2h mobs.

There are a few ways to get actual delay data. For example figuring out the tp gained per hit. Or as @WinterSolstice8 tells me frame counts but that's a little over my head atm so I've tagged him if he wants to explain it.

I hope it doesn't come off as nitpicky about the mobs delay ratings.

Reply:
It's not nitpicky. If there is a better more concrete way to get cmbdelays by all means. I understand there are base delays for weapons/mobs but most NMs seem to not mimic these. They seem to be faster in most cases.

Unfortunately I don't have the capability to get tp gained/hit info as most of the PRs I would be submitting I can't pop on my retail character anymore and I am not sure how to count frame rates. I would need an explanation on how to get all this information so I can accurately provide a concrete delay in these cases.

I can leave -- todo on all the delays for future PRs until we come up with a really good way to do this and revisit it in the future if that ends up being the best idea.

Reply:
roll back all 3 delays. We'll figure out the delays and adjust it on our side.

I thought you already had when you resolved this conversation

Reply:
I can push the last changes myself if thats ok by you

Reply:
I'm going to go ahead and hope you're cool with it